assets/scripts/storage.js

```diff
@@ -0,0 +1,53 @@
+// Saves new goals
+function save() {
```

Comment:
Let's change the function name, save() is pretty generic and if we make another save function later without thinking that could break some things

Reply:
Resolved